app/src/pages/nanomine/tools/IntelligentCharacterize.vue

```diff
@@ -1,14 +1,89 @@
 <template>
-  <tool-template :tool="tool" :card="card"></tool-template>
+  <tool-template :card="card" name="IntelligentCharacterize" :job="job" header="Intelligent Characterization"
+    :reference-list="references">
+    <!-- card-specific slots -->
+    <template #image>
+      <img src="@/assets/img/nanomine/Intelligent_Characterization.png" alt="Intelligent Characterization">
+    </template>
+    <template #actions>
+      <router-link to="IntelligentCharacterize">
+        <md-button class="md-raised md-primary md-raised">
+          Use Intelligent Characterization Webtool
+        </md-button>
+      </router-link>
+    </template>
+    <!-- shared slots -->
+    <template #title>
+      Intelligent Characterization
+    </template>
+    <template #content>
+      The intelligent characterization tool selects the most suitable characterization method between the “physical
+      descriptors” and the “spectral density function (SDF)” approaches based on analyzing the user uploaded image(s).
+      Results generated can be easily passed to the NanoMine Database.
+    </template>
+    <!-- tool page-specific slots-->
+    <template #description>
+      This tool assesses the uploaded image(s) of microstructure and performs analysis to decide which characterization
+      method suits the best. The process consists of three steps:
+      <br>
+      <ol>
+        <li>Calculate universal descriptors, i.e., void fraction, isotropy index, and interfacial area.</li>
+        <li> Determine whether Spheroidal Descriptors or Spectral Density Function (SDF) is more applicable, based on
+          isotropy index and connectivity index. If the material is not isotropic or too connected, then the Spheroidal
+          Descriptors are not applicable.</li>
+        <li>Generate results for either Spheroidal Descriptors or SDF representation. If Spheroidal Descriptor method is
+          chosen, results of Nearest Neighbor Distance, Number of clusters, Compactness, Cluster Area, Cluster Radius,
+          Elongation Ratio, Orientation Angle, and Rectangularity will be provided. If the SDF method is chosen, the
+          following five different forms of SDF will be tested and the best fitted form will be chosen with output of
+          the fitted parameters and the goodness of fit value.</li>
+      </ol>
+      <br>
+      Chi-square fit: <math>y = a * f<sub>k</sub>(b * x, n)</math>, where <math>f<sub>k</sub></math> is the probability
```

Comment:
I agree that MathJax seems like the better option

Reply:
Added it in 441e170, I'll bring it up at the standup today.

Reply:
@tholulomo

Reply:
bbaffd7 is the updated and fixed MathML syntax, which works in Firefox by default but not Chrome, for example. 441e170 includes the MathJax CDN and the `vue-mathjax` component, which works in more browsers at the cost of including the CDN. I looked for ways to include a CDN script tag on only one component, but didn't find anything.

Reply:
I see. I believe we should be good since it's required in a single component and that component and its resources don't load up (by default) until it is called anyway.